apps/www/components/Questionnaire/TextQuestion.js

```diff
@@ -47,13 +47,15 @@ class TextQuestion extends Component {
 
   render() {
     const {
-      question: { text, explanation, maxLength },
+      question: { text, explanation, maxLength, order, metadata },
       t,
     } = this.props
     const { value } = this.state
+    const beforeP = metadata && metadata.beforeP
     return (
       <div>
-        {text && <H2 {...questionStyles.label}>{text}</H2>}
+        {beforeP && <Editorial.Subhead>{beforeP}</Editorial.Subhead>}
```

Comment:
Will we understand what `beforeP` does next year still? 😅

Reply:
oh ye of little faith

Reply:
renaming to `questionHeader`